app/services/idv/session.rb

```diff
@@ -169,9 +168,25 @@ def failed_phone_step_numbers
       session[:failed_phone_step_params] ||= []
     end
 
+    def pii_from_doc=(new_pii_from_doc)
+      if new_pii_from_doc.blank?
+        session[:pii_from_doc] = nil
+      else
+        session[:pii_from_doc] = new_pii_from_doc.to_h
+      end
+    end
+
+    def pii_from_doc
+      return nil if session[:pii_from_doc].blank?
+      Pii::StateId.new(**session[:pii_from_doc])
```

Comment:
I think this should filter the keys, because if we ever add new keys to this struct this will error during in a +1/-1 (50-50) state [(prior art here)](https://github.com/18F/identity-idp/blob/5c758fd167df84170312ec58ab8daba842d4c730/app/services/pii/attributes.rb#L27)

```suggestion
      Pii::StateId.new(**session[:pii_from_doc].slice(*Pii::StateId.members))
```